### PR TITLE
Gogs merge: Realtime webhooks

### PR DESCRIPTION
**Pull request by tboerger**
(cherry picked from commit fa298a2c30c358dbfa47fc123c6aca83fe9eb999)

### CODE REVIEW

**Comment by vtolstov** [comment]
please fix pr and merge

**Comment by tboerger** [comment]
I tried to port it as good as possible, needs to be tested!

**Comment by vtolstov** [comment]
I'm test it now

**Comment by lunny** [comment]
Is this pr OK? @tboerger @vtolstov

**Comment by vtolstov** [comment]
In my case all good.
31 июля 2015 г. 5:06 пользователь "Lunny Xiao" <notifications@github.com>
написал:

File: Makefile [hunk 1]
```diff
@@ -74,6 +74,10 @@ build: $(BIN)/$(EXECUTABLE)
 
 $(BIN)/$(EXECUTABLE): $(wildcard *.go)
 	go build -v -tags '$(TAGS)' -ldflags '-s -w $(LDFLAGS)' -o $@
+	cp -r ./templates ./bin/
+	cp -r ./public ./bin/
+	cp -r ./conf ./bin/
+	cp -r ./scripts ./bin/
```
[the remark this answers is not in the document]
Comment (tboerger) [reply Makefile:80]: Because scripts got only stuff like init scripts included which is not required for that.

**Pull request merged**